lux/_config/config.py

```diff
@@ -0,0 +1,109 @@
+'''
+This config file was largely borrowed from Pandas config.py set_action functionality.
+For more resources, see https://github.com/pandas-dev/pandas/blob/master/pandas/_config
+'''
+from collections import namedtuple
+from typing import Any, Callable, Dict, Iterable, List, Optional
+
+RegisteredOption = namedtuple("RegisteredOption", "key function validator args")
+
+# holds registered option metadata
+_registered_actions: Dict[str, RegisteredOption] = {}
+update_actions: Dict[str, bool] = {}
+update_actions["flag"] = False
```

Comment:
Can you add a comment to explain what `update_actions` does?

Reply:
Still unclear what `update_actions` does, I think it checks if the actions are updated? Adding a line comment here would be useful.

Reply:
my bad I had to revert some comments during editing to test ill add it back